pkg/procedure/procedure_test.go

```diff
@@ -51,3 +52,36 @@ func TestProcedureNameAndSplit(t *testing.T) {
 		assert.Equal(t, tt.Method, m)
 	}
 }
+
+func BenchmarkToName(b *testing.B) {
+	benchmarks := []struct {
+		desc         string
+		stringLength int
+	}{
+		{"StringsLength-10", 10},
+		{"StringsLength-100", 100},
+		{"StringsLength-1000", 1000},
```

Comment:
I think this is a bit flaky? With 10, we're likely just randomizing enough of the same stringsto get to "0" ? Otherwise the numbers don't seem to make sense.

```
name                          old allocs/op  new allocs/op  delta
ToName/StringsLength-10-10        3.00 ± 0%      0.00       -100.00%  (p=0.000 n=100+100)
ToName/StringsLength-100-10       3.00 ± 0%      1.00 ± 0%   -66.67%  (p=0.000 n=100+100)
ToName/StringsLength-1000-10      3.00 ± 0%      1.00 ± 0%   -66.67%  (p=0.000 n=100+100)
```

I would just put a single basic benchmark with a hardcoded value?

Reply:
Agreed, I will update the benchmark strategy and revise the summary accordingly.